torch/csrc/jit/script/init.cpp

```diff
@@ -534,9 +562,10 @@ void initJitScriptBindings(PyObject* module) {
     .def("graph_for", [](Method& self, py::args args, py::kwargs kwargs) {
       return self.graph_for(createStackForSchema(self.getSchema(), std::move(args), std::move(kwargs)));
     })
-    .def("forward_schema", [](Method &self, Def &def, bool is_method) {
-      auto schema = extractSchemaFromDef(def, is_method);
-      self.setSchema(schema);
+    .def("forward_schema", [](Method &self, Def &def, py::object defss, bool is_method) {
```

Comment:
same nit as last review:
```
(Method &self, Def &def, FunctionDefaults defaults, bool is_method)
```
enables you to remove the cast, and will improve the error message if this is called incorrectly.